docs/rfcs/0002/readme.md

```diff
@@ -37,21 +34,58 @@ The proposal below might take a couple of read-throughs; I've also added a concr
-5. The USER submits a WLM job with the #DW container fields populated
-6. WLM runs the job and drives the job through the following stages...
-    1. Proposal: RABBIT validates the #DW container directive by comparing the supplied values to what is listed in the NNF Container Profile. If the USER fails to meet the requirements, the job fails. 
-    2. Pre-run: RABBIT software will:
-        1. create a config map reflecting the storage requirements and any runtime parameters; this is provided to the container at the volume mount named "nnf-config", if specified.
-        2. duplicate the pod template specification from the Container Profile and patches the necessary Volumes and the config map. The spec is used as the basis for starting the necessary pods and containers.
-    3. The containerized application executes. The expected mounts are available per the requirements and celebration occurs.
-
-Example
--------
-
-Say I authored a simple application, `foo`, that requires Rabbit local GFS2 storage and a persistent Lustre storage volume. As the author, my program is coded to expect the GFS2 volume is mounted at `/foo/local` and the Lustre volume is mounted at `/foo/persistent`. In this case, the storages are not optional, so they are defined as such in the NNF Container Profile.
+    3. the NNF storage requirements described above.
+3. The ADMINISTRATOR creates a corresponding _NNF Container Profile_ Kubernetes custom resource with the necessary NNF storage requirements and pod specification as described by the AUTHOR
+4. The USER who desires to use the application works with the AUTHOR and the related NNF Container Profile to understand the storage requirements
+5. The USER submits a WLM job with the #DW container directive variables populated
+6. WLM runs the workflow and drives it through the following stages...
+    1. `Proposal`: RABBIT validates the #DW container directive by comparing the supplied values to those listed in the NNF Container Profile. If the workflow fails to meet the requirements, the job fails
+    2. `PreRun`: RABBIT software:
+        1. creates a config map reflecting the storage requirements and any runtime parameters; this is provided to the container at the volume mount named `nnf-config`, if specified
+        2. duplicates the pod template specification from the Container Profile and patches the necessary Volumes and the config map. The spec is used as the basis for starting the necessary pods and containers.
+    3. The containerized application(s) executes. The expected mounts are available per the requirements and celebration occurs. The pods continue to run until:
+       1. the pod completes successfully (any failed pods will be restarted)
+       2. timeout (i.e. `activeDeadlineSeconds`) is hit (optional)
+       3. the max number of pod retries (i.e. `backoffLimit`) is hit (indicating failure on all retry attempts)
```

Comment:
I don't know if I'd mention all these details in a simple high-level overview. You should use links and references where the details can be further explained
For example: "The application will execute with crashes and failures handled per the \[Restart Policy\](link to Restart Policy)"

Reply:
I think these are pretty important details to consider because we have to wrap up all these conditions into deciding if the PostRun stage should go to `Ready:true` or not. Isn't that the 1 mechanism we have to Flux to indicate the success of the containers?

Reply:
I'm not saying "dont mention them." I'm saying structure your document so details and knowledge evolve from trivial examples to more select use cases and specialization